int32[] distanceData
int32 numLaser
EncodersData position
